src/App.js

```diff
@@ -1,12 +1,24 @@
 import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 
 import { history } from "helpers/history";
+import LoginPage from "pages/login-page/LoginPage";
 import HomePage from "pages/home-page/HomePage";
+import { loginPageLink, homePageLink } from "constants/linkConstants";
+import PrivateRoute from "helpers/PrivateRoute";
 
 const App = () => (
   <Router history={history}>
     <Routes>
-      <Route exact path="/" element={<HomePage />} />
+      <Route exact path={loginPageLink} element={<LoginPage />} />
```

Comment:
Instead of writing every <Route> manually, WDYT if you create an array of objects for every route just like: 
```
const routes = [
  {
    path: '/home',
    component: <Home />,
    private: true,
    exact: true,
  },
  {
    path: RoutherPaths.LOG_IN,
    component: <LogIn />,
  },
  {
    path: RoutherPaths.SIGN_UP,
    component: <SignUp />,
  }
];
```
And then you can map it just like this:
```
  {routes.map((route) => (
         <Route
              key={route.path}
              path={route.path}
              element={(
                <PrivateRoute>
                  {route.component}
                </PrivateRoute>
            />
     )}
```